CoorpAcademy/mobile
# Fetch chapters bundle

## Detailed purpose of the PR

This PR adds chapters bundle fetch from cards.

Trello: https://trello.com/c/3hAJP6Ar

### Result and observation

**Bundle request**:

![Capture d’écran 2019-05-17 à 12 43 28](https://user-images.githubusercontent.com/8419208/57923068-8b152f80-78a1-11e9-9364-895a0e95049d.png)

**Bundle success**:

![Capture d’écran 2019-05-17 à 12 43 53](https://user-images.githubusercontent.com/8419208/57923076-8ea8b680-78a1-11e9-8400-51fe1b1aca35.png)

**Bundle state**:

![Capture d’écran 2019-05-17 à 12 44 32](https://user-images.githubusercontent.com/8419208/57923078-91a3a700-78a1-11e9-87af-677e3ea3f840.png)

**Local storage**:

![Capture d’écran 2019-05-17 à 12 41 33](https://user-images.githubusercontent.com/8419208/57922903-22c64e00-78a1-11e9-9cdf-744efb7b2ab4.png)

- [ ] **Extra dependency ?**

### How to test

1. Add a microlearning course to your wishlist on the MOOC.
2. Sign in with your account (the same platform used in 1st step)
3. Look at your redux actions/state and local storage in console
4. You should be able to retrieve the data from the slides player for microlearning courses

### Testing Strategy

- [ ] Already covered by tests
